### DISCUSSION TITLE
Both branches (`dgus-reloaded` being the older firmware, previously on the `master` branch) are now up-to-date.

### DISCUSSION THREAD

**Discussion by jacobcd** [Desuuuu/klipper#106] [General]
Are you aware of any issues with 

`git checkout desuuuuklipper/master`

I seem to get this error now:

_error: pathspec 'desuuuuklipper/master' did not match any file(s) known to git_

Both branches (`dgus-reloaded` being the older firmware, previously on the `master` branch) are now up-to-date.

MCU code compiles but I haven't really tested Klippy for now. I'll be testing the `dgus-display` branch soon.

I don't plan on keeping support for the `dgus-reloaded` branch for much longer, so I suggest moving on to the newer version if possible.

_Originally posted by @Desuuuu in https://github.com/Desuuuu/klipper/discussions/103#discussioncomment-1870549_

**Comment by Desuuuu** [comment]
Run this to switch from `master` to `dgus-reloaded`: `git fetch && git checkout dgus-reloaded`

Or, if your remote for this repository is named `desuuuuklipper`: `git fetch && git checkout desuuuuklipper/dgus-reloaded`

**Reply by jacobcd** [reply]
thank you.

**Reply by jacobcd** [reply]
@Desuuuu i dropped you some BTC keep up the good work.

**Reply by Desuuuu** [reply]
Thank you!
